conda_package/mpas_tools/mesh/creation/sort_mesh.py

```diff
@@ -0,0 +1,187 @@
+
+import numpy as np
+import os
+import xarray
+import argparse
+from scipy.sparse import csr_matrix
+from scipy.sparse.csgraph import reverse_cuthill_mckee
+
+
+def sort_fwd(data, fwd):
+    vals = data.values
+    vals = vals[fwd - 1]
+    return vals
+
+
+def sort_rev(data, rev):
```

Comment:
```suggestion
def _sort_rev(data, rev):
```
Make this private, move it below the "public" functions and add a brief docstring.